byu_pytest_utils/dialog.py

```diff
@@ -213,7 +213,7 @@ def _score_output(
 
     for exp_file, obs_file in expected_files:
         if not obs_file.exists():
-            obs_content = f'File not found: {obs_file}. Did you write it?'
+            obs_content = f'File not found: {obs_file}. Did you write it?\n' + observed_io
```

Comment:
Because this line already uses a formatted string, we should stay consistent:
```python
obs_content = f'File not found: {obs_file}. Did you write it?\n{observed_io}'
```